### PR TITLE
Feat/alias/config ccs statistics

### PR DESCRIPTION
**Pull request by JeremyDec**
## Statistics alias contact center solutions new page
MANAGER-987
### Description of the Change
New Statistics page for Contact Center Solution numbers.

Two commits :

bc0fd84 : moving necessary files
02e91da : New page

### CODE REVIEW

File: client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/statistics/telecom-telephony-alias-configuration-statistics.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            <dd>
+                <tuc-elapsed-time data-ng-if="$ctrl.getMaxWaitTime()"
+                                  data-from="$ctrl.getMaxWaitTime()"></tuc-elapsed-time>
+                <span data-ng-if="!$ctrl.getMaxWaitTime()">-</span>
+            </dd>
+        </dl>
+
+        <oui-datagrid class="my-5"
+                      data-rows="$ctrl.calls">
+            <oui-column data-title="'telephony_alias_configuration_statistics_calls_calling_number' | translate"
```
Comment (marie-j) [inline client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/statistics/telecom-telephony-alias-configuration-statistics.html]:
```suggestion
            <oui-column data-title=":: 'telephony_alias_configuration_statistics_calls_calling_number' | translate"
```
?

File: client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/statistics/telecom-telephony-alias-configuration-statistics.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            </oui-column>
+            <oui-column data-title="'telephony_alias_configuration_statistics_calls_queue_time' | translate"
+                        data-property="begin">
+                <tuc-elapsed-time data-from="$row.begin"></tuc-elapsed-time>
+            </oui-column>
+            <oui-column data-title="'telephony_alias_configuration_statistics_calls_time' | translate"
+                        data-property="answered">
+                <span class="text-capitalize"
+                      data-ng-if="!$row.answered"
+                      data-ng-bind="$row.answered ? 'yes' : 'no' | translate"></span>
```
Comment (marie-j) [inline client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/statistics/telecom-telephony-alias-configuration-statistics.html]: the `ng-bind` will always evaluate to the `no` translation when displayed

File: client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/statistics/telecom-telephony-alias-configuration-statistics.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            <oui-action-menu data-align="end" data-compact>
+                <oui-action-menu-item data-on-click="$ctrl.actionOnCall($row, 'intercept')"
+                                      data-ng-if="!$row.answered">
+                    <span data-translate="telephony_alias_configuration_mode_calls_action_intercept"></span>
+                </oui-action-menu-item>
+                <oui-action-menu-item data-on-click="$ctrl.actionOnCall($row, 'eavesdrop')"
+                                      data-ng-if="$row.answered">
+                    <span data-translate="telephony_alias_configuration_mode_calls_action_eavesdrop"></span>
+                </oui-action-menu-item>
+                <oui-action-menu-item data-on-click="$ctrl.actionOnCall($row, 'transfert')"
```
Comment (marie-j) [inline client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/statistics/telecom-telephony-alias-configuration-statistics.html]:
```suggestion
                <oui-action-menu-item data-on-click="$ctrl.actionOnCall($row, 'transfer')"
```
no `t` in english 😉

File: client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/statistics/telecom-telephony-alias-configuration-statistics.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    <oui-spinner></oui-spinner>
+</div>
+
+<div class="row"
+     data-ng-if="!$ctrl.loading">
+
+    <div class="col-lg-8">
+        <h2 data-translate="telephony_alias_configuration_statistics_title"></h2>
+        <p data-translate="telephony_alias_configuration_statistics_subtitle"></p>
+        <small data-translate="telephony_alias_configuration_statistics_subtitle_help"></small>
```
Comment (FredericEspiau) [inline client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/statistics/telecom-telephony-alias-configuration-statistics.html]: `small` is for fine prits

File: client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/statistics/telecom-telephony-alias-configuration-statistics.html (hunk 2 above)
Comment (FredericEspiau) [reply client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/statistics/telecom-telephony-alias-configuration-statistics.html]: Also, do you have a `yes` and `no` translation ids ?

File: client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js [hunk 5]
```diff
@@ -230,6 +230,60 @@ export default class {
...
+  fetchContactCenterSolutionNumberQueueCalls({ billingAccount, serviceName }, queueId) {
+    return this.OvhApiTelephony.EasyHunting().Hunting().Queue().LiveCalls()
+      .v6()
+      .query({
+        billingAccount,
+        serviceName,
+        queueId,
+      }).$promise
+      .then(callsIds => this.$q
+        .all(callsIds.reverse().map(
```
Comment (FredericEspiau) [inline client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js]: Please check that `callsIds` is not a string

File: client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js [hunk 6]
```diff
@@ -402,6 +456,41 @@ export default class {
...
+  fetchContactCenterSolutionNumberAgentsStatus({ billingAccount, serviceName }, queueId) {
+    return this.OvhApiTelephony.EasyHunting().Hunting().Queue().Agent()
+      .v6()
+      .query({
+        billingAccount,
+        serviceName,
+        queueId,
+      }).$promise
+      .then(agentIds => this.$q
+        .all(agentIds.map(
```
Comment (FredericEspiau) [inline client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js]: `typeof agentIds  !== 'string'`

File: client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/statistics/telecom-telephony-alias-configuration-statistics.html (hunk 2 above)
Comment (JeremyDec) [reply client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/statistics/telecom-telephony-alias-configuration-statistics.html]: @FredericEspiau  For sure => https://github.com/ovh-ux/ovh-manager-telecom/blob/master/client/app/common/translations/Messages_fr_FR.xml#L72

File: client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js (hunk 5 above)
Comment (JeremyDec) [reply client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js]: Why ? query request are defined by `isArray: true` (see: https://github.com/ovh-ux/ovh-api-services/blob/master/src/telephony/easyHunting/hunting/queue/liveCalls/telephony-easyHunting-hunting-queue-liveCalls.v6.service.js#L12)

File: client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js (hunk 6 above)
Comment (JeremyDec) [reply client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js]: Same than callsIds query method

File: client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js (hunk 5 above)
Comment (FredericEspiau) [reply client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js]: `string` is an `array`: `['e', 'r', 'r', 'o', 'r']`

We've had issues with the API sending us error in strings and I'm not sure `isArray: true` prevents this issue (deserves a try though)

**Pull request merged**